plugins/bnet_plugin/bnet_plugin.cpp

```diff
@@ -482,6 +485,12 @@ namespace eosio {
               idx.erase(itr);
               itr = idx.begin();
            }
+
+           auto bitr = _block_status.find(s->id);
```

Comment:
can we put this inside the branch on `_remote_request_irreversible_only` to avoid the cost of the lookup for all other types of connections?